monai/networks/blocks/patchembedding.py

```
@@ -29,6 +29,32 @@
+class _PatchRearrange(nn.Module):
+    """Fallback patch rearrangement using pure PyTorch, for einops compatibility."""
+
+    def __init__(self, spatial_dims: int, patch_size: tuple) -> None:
+        super().__init__()
+        self.spatial_dims = spatial_dims
+        self.patch_size = patch_size
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        B, C = x.shape[0], x.shape[1]
+        sp = x.shape[2:]
+        g = tuple(s // p for s, p in zip(sp, self.patch_size))
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)